### PR TITLE
tests: Use testTmpDir var to specify tmp directory

### PR DESCRIPTION
**Pull request by vadmeste**
## Description
To be able to specify the directory where tests will be done.  This way,
it will be easier to run Minio tests on a mounted directory like NFS, ..

## Motivation and Context
Testing go test on NFS

## How Has This Been Tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: cmd/test-utils_test.go [hunk 1]
```diff
@@ -133,6 +133,9 @@ const (
 var randN uint32
 var randmu sync.Mutex
 
+// Temp files created in default Tmp dir
+var testTmpDir = os.TempDir()
```
Comment (harshavardhana) [inline cmd/test-utils_test.go]: name this as 'globalTestTmpDir'

**Review by harshavardhana — CHANGES_REQUESTED** [review]
How do you specify NFS directory?

**Comment by vadmeste** [comment]
> How do you specify NFS directory?

Currently no way, I am thinking of adding a function which returns the content of an environment variable if non nil or default value

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
Don't bring any new environment variables. See if go test can take custom cli options.

**Pull request merged**